qutip/solver/heom/bofin_solvers.py

```diff
@@ -105,6 +105,7 @@ class HierarchyADOs:
     labels: list of tuples
         A list of the ADO labels within the hierarchy.
     """
+
```

Comment:
Could you revert all of these additional newlines? There is no need for this PR to touch this file at all.